benchmarks/user/make-pbs.py

```diff
@@ -0,0 +1,74 @@
+import os
```

Comment:
Is this script tested somewhere?

Reply:
no. Use it as a template/support tool -- that's the idea (see also updated README)